R/commonbayesianttest.R

```diff
@@ -644,9 +645,10 @@
             mean <- .clean(mean(groupDataOm))
             std <- .clean(sd(groupDataOm))
             sem <- .clean(sd(groupDataOm) / sqrt(length(groupDataOm)))
+            cov  <- .clean(sd(groupDataOm) / abs(mean(groupDataOm)))
```

Comment:
Can you pick one name for the coefficient and use it throughout? I see in your code using `cov`, `cv`, etc., it would be good to use only one of those variable names. Also pick something else than `cov` because it can get confused with "covariance". Note, there is nothing wrong about long but descriptive names, so do not be shy to use something like `coefOfVariation`.

Reply:
OK, I'll adjust this! coefOfVariation sounds good!

Reply:
Also, you can actually get rid of the `.clean()`, because that should be now handled by the jaspTable automatically.

Reply:
OK, will do!